File: src/__tests__/fixtures/python/sample.py
"""Module docstring."""
import os
from collections import defaultdict

# A simple Calculator
class Calculator:
    """A basic calculator class."""

    def add(self, a, b):
        """Add two numbers."""
        return a + b

    def subtract(self, a, b):
        return a - b


def multiply(x, y):
    """Multiply two numbers."""
    return x * y


@staticmethod
def helper():
    return 42

File: src/__tests__/fixtures/typescript/sample.ts
/** A simple Calculator */
class Calculator {
  add(a: number, b: number): number {
    return a + b;
  }
}

function multiply(x: number, y: number): number {
  return x * y;
}

interface Shape {
  area(): number;
}

type Point = { x: number; y: number };

export { Calculator, multiply };
